include/flatbuffers/flatbuffers.h

```diff
@@ -1284,9 +1289,17 @@ class Verifier FLATBUFFERS_FINAL_CLASS {
     return true;
   }
 
+  // Returns the message size in bytes
+  size_t GetSize() const {
+    uintptr_t sz = upper_bound_ ? upper_bound_ - buf_  : 0;
+    sz = (sz - 1 + 4) & -uintptr_t(4);
+    return sz;
+  }
+
  private:
   const uint8_t *buf_;
   const uint8_t *end_;
+  mutable const uint8_t *upper_bound_;
```

Comment:
It makes more sense to remove const where needed, since Verifier objects are typically never const to start with.